ops/start-iframe-app.sh

```diff
@@ -14,6 +14,14 @@ then interactive=(--interactive --tty)
 else echo "Running in non-interactive mode"
 fi
 
+####################
+# Misc Config
+
+# Load the config with defaults if it does not exist
+if [[ ! -f "$root/browser.config.json" ]]
```

Comment:
I think you can do -z instead of the ! -f. We do that elsewhere.

Reply:
ahh i copy-pasted from `start-trio`, so i'm gonna leave it